### DISCUSSION TITLE
希望新增一个去掉首页赛事的功能

### DISCUSSION THREAD

**Discussion by Index-z** [the1812/Bilibili-Evolved#3091] [综合讨论 / General]
首页分区排序无法对赛事进行调整，但是赛事始终显示在最上面，就很苦恼。

**Comment by Geeyun-JY3** [comment]
v2 的 `简化首页` 就是干这个的，不过新版首页出来后这功能好像没适配过。

**Reply by Index-z** [reply]
用过不太好用 我觉的B站现在首页就挺好 就是赛事比较烦人总在最上面导致必须网页往下翻才能看到想看的。

**Reply by Geeyun-JY3** [reply]
我不是说那个完全替换掉 b 站原有首页的功能，v2 的 `简化首页` 就是你要的屏蔽首页分区功能，这个功能一直都有，只是在 v1 不叫这个名字；另外 v2 的 `简化首页` 目前好像还没适配现在这个已经全量了的新首页。

**Reply by Index-z** [reply]
看到了有个首页过滤的功能，完美解决了我想弄掉首页赛事的想法。太棒了！！！
